Resources/Prototypes/Entities/Objects/Consumable/Smokeables/Cigarettes/packs.yml

```
@@ -0,0 +1,87 @@
+- type: entity
+  id: CigPackBase
+  parent: BaseItem
+  name: cigarette pack
+  abstract: true
+  components:
+    - type: Sprite
+      state: closed
```

```suggestion
      state: closed
      netsync: false
```

This should solve your issues with cigars not closing. Open/Close bag visualizations are client only.